db/schema.rb

```diff
@@ -953,6 +952,24 @@
     t.index ["person_id"], name: "fk_persons"
   end
 
+  create_table "records", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
The table seems to have "moved" in the schema definition file, because the name mysteriously changed?!